apis/v1alpha1/instrumentation_types.go

```diff
@@ -25,10 +25,10 @@ type InstrumentationSpec struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	Exporter `json:"exporter,omitempty"`
 
-	// ResourceAttributes defines attributes that are added to resource.
+	// Resource defines resource configuration
```

Comment:
I'm sure you can come up with a better doc for this ;-)

Reply:
any suggestions to naming are welcome

Reply:
Sure: "Resource defines the configuration for the resource attributes, as defined by the OpenTelemetry SDK. See also: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/overview.md".

Remember that folks using this CR might not be that familiar with OpenTelemetry *yet*, so, the word "Resource" might be confusing if they don't have the full context.